src/types.ts

```diff
@@ -0,0 +1,23 @@
+declare module "cli-git.io";
+
+interface Rule {
+  heading: string;
+  docs: string;
+  suggestion: (value?: number | string) => string;
+  rationale: string;
+  source: string[];
+  ok: string;
+  notOk: string;
+  listen?: string;
+  rules?: string[];
+  fn?: (item: string, alt: string) => string | boolean;
+  id?: string;
+}
```

Comment:
💡 This interface defines the shape for each rule, including key names, type, and whether it's optional (`?`). As a result, this PR is able to remove some unit tests that TypeScript made redundant in clues.test.js. Ref 4419555.